### ISSUE TITLE
Create a landing page EF (use XD, etc.) in order to break down the components

### ISSUE THREAD

**Issue by kevanantha** [evilfactorylabs/edan#1]

**Comment by ri7nz** [comment]
First, follow the guideline here https://github.com/evilfactorylabsarchive/design#colors

**Comment by kevanantha** [comment]
Okay @ri7nz

**Comment by faultable** [comment]
good job gengs

**Comment by kevanantha** [comment]
BTW, I just read these articles;
https://www.invisionapp.com/inside-design/guide-to-design-systems/
https://medium.muz.li/how-to-build-design-systems-3431560f51fb
https://www.figma.com/blog/how-to-build-your-design-system-in-figma/
https://www.invisionapp.com/talks/design-systems-zero-to-one

The way how we build design system by creating the landing page example is a bit wrong (I think)

Those articles tell us to build design system with [atomic design principles](http://atomicdesign.bradfrost.com/)

Should we follow their steps? @ri7nz @faultable

**Comment by ri7nz** [comment]
TLDR; yeah we can follow.

**Comment by kevanantha** [comment]
I think, we have to split up the issues e.g for button, alert, etc.

wdyt? @ri7nz @faultable

**Comment by ri7nz** [comment]
Make 1 design for landing page, then we can split for button, alert, etc. 

1 design landing page is 1 goal or milestone.

**Comment by kevanantha** [comment]
Well, okay then

**Comment by ri7nz** [comment]
Any update

**Comment by faultable** [comment]
How the project is going on? Any blockers?

**Comment by kevanantha** [comment]
I leave sick for 5 days, can't do anything
will continue ASAP

blockers: Figma can't share components unless premium, so I will migrate to XD

**Comment by ri7nz** [comment]
@kevanantha Nope, Take a rest bruh, I leave sick 3 days & do nothing to. 

Good job :rocket:
